app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,18 @@
+class CommentsController < ApplicationController
+  def create
+    @comment = Comment.new(comment_params)
+    @comment.post = Post.find(params[:post_id])
+    @comment.author_id = current_user.id
+    redirect_to user_post_path(id: @comment.post_id) if @comment.save
+  end
```

Comment:
- [ ] @Fahim-Khan-P Great work on your create method, however it's good practice to have a redirect path for when a comment is created and when it fails, currently you only have a path for when its created.

Reply:
Thank you for your feedback! I agree that it's good practice to have a redirect route for both success and failure when creating a comment. I have updated the create method in the **CommentsController** to handle both cases: